pages/design/rule-design.md

```diff
@@ -39,34 +79,64 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 Other aspects may be necessary for testing. These can be added as long as they are sufficiently defined.
 
+For more details, see [ACT Rules Format: Input](https://www.w3.org/TR/act-rules-format/#input).
+
 ### Authors
 
-Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`.
+Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`. Note: If an author does not exist in `package.json`, they should be added to that file as part of the relevant PR.
+
+## Applicability
+
+Applicability describes which (elements of) web pages should be tested using the rule. These elements are known as test targets. Applicability must be written in plain language, as well-formed grammatically correct sentences, so that it can be used by QA testers. Applicability must rely on well defined properties of the technologies that are tested. For instance, a rule may be applicable to all `video` elements, but it can not be applicable to all `object` elements used to show video, unless the term "video" is further defined.
+
+Finding objective definitions to use in rules can be difficult, if not outright impossible in some cases. The intent here is to ensure repeatability of the rule. Not everything in WCAG testing is entirely repeatable, but when it comes to rule applicability, this is a hard requirement.
+
+> _For example:_ A rule testing button accessible names could apply to elements other than the `button` element and this could be stated as _"The rule applies to elements that are included in the accessibility tree with the semantic role of button, except for `input` elements of `type="image"`."_.
```

Comment:
```suggestion
> _For example:_ A rule testing button accessible names could apply to elements other than the `button` element and this could be stated as _"This rule applies to elements that are included in the accessibility tree with the semantic role of button, except for `input` elements of `type="image"`."_.
```

Reply:
Wondering if I should use a different example, as this particular pair of rules is currently under discussion and may change.

Reply:
I agree, it's fairly easy to pick just any other applicability 😄

Reply:
```suggestion
> _For example:_ A rule testing that page titles are descriptive should only apply to specific `title` elements and this could be stated as _"This rule applies to the first HTML title element that is a descendant of the html element of a web page, and
contains children that are text nodes that are not only whitespace."_.
```